.npmignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
From https://docs.npmjs.com/files/package.json#files: 

>You can also provide a .npmignore file in the root of your package or in subdirectories, which will keep files from being included. At the root of your package it will not override the “files” field, but in subdirectories it will

As I saw one of these in our 1.9.1 upload, https://unpkg.com/browse/mapbox-gl@1.9.1/src/.